server/src/main/java/org/elasticsearch/cluster/coordination/StableMasterHealthIndicatorService.java

```diff
@@ -269,9 +270,8 @@ private List<UserAction> getContactSupportUserActions(boolean explain) {
         if (explain) {
             UserAction.Definition contactSupport = new UserAction.Definition(
                 "contact_support",
-                "The Elasticsearch cluster does not have a stable master node. Please contact Elastic Support "
-                    + "(https://support.elastic.co) to discuss available options.",
-                null
+                "The Elasticsearch cluster does not have a stable master node. Get help at " + GET_HELP_GUIDE,
```

Comment:
I wonder if we can point users to this page as well? 
```suggestion
                "The Elasticsearch cluster does not have a stable master node. You may need to scale up the master node based on sizing guidance. Need more help? " + GET_HELP_GUIDE,
```
Can you also link the scale up the master node to this page? https://www.elastic.co/guide/en/elasticsearch/reference/master/size-your-shards.html#shard-count-recommendation